src/lib.rs

```diff
@@ -41,70 +116,77 @@ bitflags::bitflags! {
         /// The permission to send messages to this capability.
         const SEND = 1 << 0;
 
-        /// The permission to link to this capability and be notified of its
+        /// The permission to monitor this capability and be notified of its
```

Comment:
We should mention somewhere that monitoring a process only goes one way and that linking (which will not be implemented in flue itself) is a superset of monitoring that is bidirectional 

this may also belong in the readme with the other stuff i mentioned